plugins/modules/proxysql_query_rules_fast_routing.py

```diff
@@ -0,0 +1,436 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: proxysql_query_rules_fast_routing
+author: "Akim Lindberg (@akimrx)"
+short_description: Modifies query rules for fast routing policies using the proxysql admin interface
+description:
+   - The M(community.proxysql.proxysql_query_rules_fast_routing) module modifies query rules for fast
+     routing policies and attributes using the proxysql admin interface.
+options:
+  username:
+    description:
+      - Filtering criteria matching username, a query will match only if the connection is made with
+        the correct username.
+    type: str
+    required: True
+  schemaname:
+    description:
+      - Filtering criteria matching schemaname, a query will match only if the connection uses
+        schemaname as its default schema.
+    type: str
+    required: True
+  flagIN:
+    description:
+      - Evaluated in the same way as I(flagIN) is in B(mysql_query_rules) and correlates to the
+        I(flagOUT/apply) specified in the B(mysql_query_rules) table.
+        (see M(community.proxysql.proxysql_query_rules)).
+    type: int
+    default: 0
+  destination_hostgroup:
+    description:
+      - Route matched queries to this hostgroup. This happens unless there is a
+        started transaction and the logged in user has
+        I(transaction_persistent) set to C(True) (refer to M(community.proxysql.proxysql_mysql_users)).
+    type: int
+    required: True
+  comment:
+    description:
+      - Free form text field, usable for a descriptive comment of the query rule.
+    type: str
+    default: ''
+  state:
+    description:
+      - When C(present), adds the rule. When C(absent), removes the rule.
+    type: str
+    choices: [ "present", "absent" ]
+    default: present
+  force_delete:
+    description:
+      - By default, we avoid deleting more than one schedule in a single batch;
+        however, if you need this behaviour and you are not concerned about the
+        schedules deleted, you can set I(force_delete) to C(True).
+    type: bool
+    default: False
+notes:
+- Supports C(check_mode).
+extends_documentation_fragment:
+- community.proxysql.proxysql.managing_config
+- community.proxysql.proxysql.connectivity
+'''
+
+EXAMPLES = '''
+---
+# This example adds a rule for fast routing
+- name: Add a rule
+  community.proxysql.proxysql_query_rules:
```

Comment:
the documentation is wrong

`proxysql_query_rules_fast_routing` not `proxysql_query_rules`

```suggestion
  community.proxysql.proxysql_query_rules_fast_routing:
```